packages/client/src/components/LoadingSpinner/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import styles from "./LoadingSpinner.module.scss";
+
+export default function LoadingSpinner() {
```

Comment:
Lol this never really loads unless you run the client independent of the server, because the turnaround on the test fetch is so quick.